source/utils/Helper.h

```diff
@@ -30,6 +30,10 @@
 #include <vector>
 #include <cstdint>
 
+#ifdef _MSC_VER
+    #define snprintf _snprintf
```

Comment:
Yes. I think we should make a choive here ^^
I think the best way would be to have a look at how other open source projects do. I will try to have a look.